core/src/impl/Kokkos_SpacePriorities.hpp

```diff
@@ -0,0 +1,98 @@
+
+#include <type_traits>
+#include <tuple>
+#include <utility>
+#include <string>
+#include <iostream>
+#include <typeinfo>
+
+#include <cxxabi.h>
+
+#ifndef __SPACE_PRIORITIES_HPP
+#define __SPACE_PRIORITIES_HPP
```

Comment:
Let's not use double underscores.
```suggestion
#ifndef KOKKOS_SPACE_PRIORITIES_HPP
#define KOKKOS_SPACE_PRIORITIES_HPP
```